cinema/models.py

```diff
@@ -12,24 +12,28 @@ class CinemaHall(models.Model):
     def capacity(self) -> int:
         return self.rows * self.seats_in_row
 
-    def __str__(self):
+    def __str__(self) -> str:
         return self.name
 
 
 class Genre(models.Model):
     name = models.CharField(max_length=255, unique=True)
 
-    def __str__(self):
+    def __str__(self) -> str:
         return self.name
 
 
 class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
-    def __str__(self):
+    def __str__(self) -> str:
         return self.first_name + " " + self.last_name
 
+    @property
+    def full_name(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
```

Comment:
These two methods do the same thing. Maybe it's better to use one of them in the other?

Reply:
In this case yes, but I thought about separate use of this both methods. If in future we want to add some new field to the actors model and override str representation (for example in admin panel etc. ` @property
    def full_name(self) -> str:
        return f"{self.first_name} {self.last_name}"

    def __str__(self) -> str:
        return self.full_name`

Reply:
hmm, agree with you. if you leave these methods independent, then they should at least be written in the same way inside (adhere to the same style)